base/markdown/parse/util.jl

```diff
@@ -11,12 +11,13 @@ macro dotimes(n, body)
 end
 
 const whitespace = " \t\r"
+const whitespace8 = map(UInt8,collect(whitespace))
```

Comment:
It's probably an unnecessary optimization. Using an array of chars is probably fast enough.